website/versioned_docs/version-5.0/api/MockResolver.md

```diff
@@ -23,21 +23,44 @@ This is useful for [storybook](../guides/storybook.md) as well as component test
 ### fixtures
 
 ```typescript
+export interface SuccessFixtureEndpoint<
+  E extends EndpointInterface = EndpointInterface,
+> {
+  endpoint: E;
+  args: Parameters<E>;
+  response: ResolveType<E>;
+  error?: false;
+}
+
+/** @deprecated */
 export interface SuccessFixture {
-  request: ReadShape<Schema, object>;
-  params: object;
+  request: FetchShape<Schema, any>;
+  params?: any;
+  body?: any;
```

Comment:
why any instead of unknown?

Reply:
Haven't tried that - but this pr is just taking what's on the maste rdocs directory and moving to 5.0 cut.